api/src/main/java/org/apache/cloudstack/api/ApiConstants.java

```diff
@@ -1020,6 +1020,16 @@ public class ApiConstants {
     public static final String PRIVATE_MTU = "privatemtu";
     public static final String MTU = "mtu";
     public static final String LIST_APIS = "listApis";
+    public static final String VM_SCHEDULE_DESCRIPTION = "description";
+    public static final String VM_SCHEDULE_ACTION = "action";
+    public static final String VM_SCHEDULE_PERIOD = "period";
+    public static final String VM_SCHEDULE_TAG = "tag";
+    public static final String VM_SCHEDULE_TIMEZONE = "timezone";
+    public static final String VM_SCHEDULE_ID = "vmscheduleid";
+
+    public static final String VM_SCHEDULE_IDS = "vmscheduleids";
```

Comment:
suggestion - such a param name isn't necessary when called for an API that's related to VM Schedule, however, this can be useful if the params are not used with VM Scheduler APIs.